dotnet/samples/TeamsChefBot/Config.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace TeamsChefBot
+{
+    public class ConfigOptions
+    {
+        public string BOT_ID { get; set; }
```

Comment:
Since Nullable is enabled. Will that cause build warnings such as `string` -> `string?`?
Fix all build warnings.

Reply:
Good point, fixed.